backend/config/telemetry.py

```
@@ -0,0 +1,30 @@
+import os
+
+from opentelemetry import trace
+from opentelemetry.exporter.otlp.proto.http.trace_exporter import OTLPSpanExporter
+from opentelemetry.instrumentation.django import DjangoInstrumentor
+from opentelemetry.sdk.trace import TracerProvider
+from opentelemetry.sdk.trace.export import BatchSpanProcessor
+
+
+def setup_telemetry():
+    # Only setup telemetry if configured to do so (e.g. in prod or staging)
+    if not os.getenv("ENABLE_OPENTELEMETRY", "False").lower() in ("true", "1", "yes"):
```

Ruff (0.15.20)

[error] Test for membership should be `not in`

Convert to `not in`

(E713)